utilities/roslz4/CMakeLists.txt

```diff
@@ -29,8 +29,8 @@ catkin_package(
 include_directories(include ${lz4_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS})
 add_library(roslz4 src/lz4s.c src/xxhash.c)
 set_source_files_properties(
-  src/lz4s.c
-PROPERTIES COMPILE_FLAGS "-Wno-sign-compare")
+  src/lz4s.c src/xxhash.c
+PROPERTIES COMPILE_FLAGS "-Wno-sign-compare" COMPILE_DEFINITIONS "XXH_NAMESPACE=ROSLZ4_")
```

Comment:
Please use a separate `set_source_files_properties` call since `-Wno-sign-compare` is not necessary for `src/xxhash.c`.